package.json

```diff
@@ -125,7 +125,8 @@
       ]
     },
     "patchedDependencies": {
-      "pkg@5.7.0": "__patches__/pkg@5.7.0.patch"
+      "pkg@5.7.0": "__patches__/pkg@5.7.0.patch",
+      "patch-package@6.5.1": "__patches__/patch-package@6.5.1.patch"
```

Comment:
we can only use patchedDependencies for dev dependencies. If we patch `patch-package`, the patch won't work when someone installs our package as a dependency, because patch-package will be not patched.

We have two options:

- either contribute a change to `patch-package` 
- or fork `patch-package`

I would prefer to contribute this to patch-package directly because maintaining a fork is additional work.

Reply:
Ah great callout @zkochan I didn't think about that, and I agree with you on the note regarding contributing to patch-package. I'll see about filing a PR there to improve the modularity!

The only problem I see arising is the difference in the patch filename naming convention (noted in [this comment](https://github.com/pnpm/pnpm/pull/5279#issuecomment-1433662604))